server/api/documents.js

```diff
@@ -7,16 +7,37 @@ router.get(
   "/",
   passport.authenticate("jwt", { session: false }),
   (req, res) => {
-    docsDb
-      .getAllDocuments()
-      .then(data => {
-        res.send(data);
-      })
-      .catch(err => {
-        console.error(err);
-        res.send(500);
-      });
+    let user = req.user.role;
+    console.log(user);
+
+    if (user) {
```

Comment:
This condition is not correct. You should use getInterpreterDocuments if the role of the user is interpreter.

Reply:
i changed to check the role